[packetbeat][windows-7-32] got some failed tests for TestParseTableRaw

## Failed Test

* **Test Name:**  `TestParseTableRaw `
* **Link:** https://github.com/elastic/beats/blob/b18b333022052d22bb515af99b926c6f58d145d4/packetbeat/procs/procs_windows_test.go#L32
* **Branch:** https://github.com/elastic/beats/pull/19803
* **Artifact Link:** https://beats-ci.elastic.co/blue/rest/organizations/jenkins/pipelines/Beats/pipelines/beats/branches/PR-19797/runs/9/nodes/2002/log/?start=0. or [gists](https://gist.github.com/v1v/dbd82646bb3b95e65f316ad6765f46af) if the build has been removed

### Stack Trace

```
[2020-10-30T10:32:52.085Z] C:\Users\jenkins\workspace\Beats_beats_PR-19797\src\github.com\elastic\beats\packetbeat>mage build unitTest 
[2020-10-30T10:33:10.213Z] >> build: Building packetbeat
[2020-10-30T10:34:31.781Z] >> go test: Unit Testing
[2020-10-30T10:35:39.544Z] FAILURES:
[2020-10-30T10:35:39.544Z] Package: github.com/elastic/beats/v7/packetbeat/procs
[2020-10-30T10:35:39.544Z] Test:    [build failed]
[2020-10-30T10:35:39.544Z] procs\procs_windows_test.go:55:68: constant 3435973836 overflows int
[2020-10-30T10:35:39.544Z] ----
[2020-10-30T10:35:39.544Z] SUMMARY:
[2020-10-30T10:35:39.544Z]   Fail:     1
[2020-10-30T10:35:39.544Z]   Skip:     0
[2020-10-30T10:35:39.544Z]   Pass:     339
[2020-10-30T10:35:39.544Z]   Packages: 24
[2020-10-30T10:35:39.544Z]   Duration: 1m9.1386719s
[2020-10-30T10:35:39.544Z]   Coverage Report: C:\Users\jenkins\workspace\Beats_beats_PR-19797\src\github.com\elastic\beats\packetbeat\build\TEST-go-unit.html
[2020-10-30T10:35:39.544Z]   JUnit Report:    C:\Users\jenkins\workspace\Beats_beats_PR-19797\src\github.com\elastic\beats\packetbeat\build\TEST-go-unit.xml
[2020-10-30T10:35:39.544Z]   Output File:     C:\Users\jenkins\workspace\Beats_beats_PR-19797\src\github.com\elastic\beats\packetbeat\build\TEST-go-unit.out
[2020-10-30T10:35:39.544Z] >> go test: Unit Test Failed
[2020-10-30T10:35:39.544Z] Error: go test failed: 1 test failures
script returned exit code 1[2020-10-30T10:35:39.897Z] 
```

#### Test output

![image](https://user-images.githubusercontent.com/2871786/97703486-9f0a1c80-1aa8-11eb-9e57-302b83ce5ee7.png)


```
# github.com/elastic/beats/v7/packetbeat/procs [github.com/elastic/beats/v7/packetbeat/procs.test]
procs\procs_windows_test.go:55:68: constant 3435973836 overflows int
```

#### XML output

```
	<testsuite tests="1" failures="1" time="0.000" name="github.com/elastic/beats/v7/packetbeat/procs">
		<properties>
			<property name="go.version" value="go1.14.7"></property>
		</properties>
		<testcase classname="procs" name="[build failed]" time="0.000">
			<failure message="Failed" type="">procs\procs_windows_test.go:55:68: constant 3435973836 overflows int</failure>
		</testcase>
	</testsuite>
```

--- a/packetbeat/Jenkinsfile.yml
+++ b/packetbeat/Jenkinsfile.yml
@@ -116,14 +116,14 @@ stages:
                 - "windows-7"
             branches: true     ## for all the branches
             tags: true         ## for all the tags
-    #windows-7-32:  See https://github.com/elastic/beats/issues/22303
-    #    mage: "mage build unitTest"
-    #    platforms:             ## override default labels in this specific stage.
-    #        - "windows-7-32-bit"
-    #    when:                  ## Override the top-level when.
-    #        comments:
-    #            - "/test packetbeat for windows-7-32"
-    #        labels:
-    #            - "windows-7-32"
-    #        branches: true     ## for all the branches
-    #        tags: true         ## for all the tags
+    windows-7-32:
+        mage: "mage build unitTest"
+        platforms:             ## override default labels in this specific stage.
+            - "windows-7-32-bit"
+        when:                  ## Override the top-level when.
+            comments:
+                - "/test packetbeat for windows-7-32"
+            labels:
+                - "windows-7-32"
+            branches: true     ## for all the branches
+            tags: true         ## for all the tags

--- a/packetbeat/procs/procs_windows_test.go
+++ b/packetbeat/procs/procs_windows_test.go
@@ -33,6 +33,7 @@ func TestParseTableRaw(t *testing.T) {
 	IPv4 := extractTCPRowOwnerPID
 	IPv6 := extractTCP6RowOwnerPID
 
+	pid := uint32(0xCCCCCCCC)
 	for idx, testCase := range []struct {
 		name     string
 		factory  extractorFactory
@@ -52,7 +53,7 @@ func TestParseTableRaw(t *testing.T) {
 			"01000000" +
 				"77777777AAAAAAAA12340000BBBBBBBBFFFF0000CCCCCCCC",
 			[]portProcMapping{
-				{endpoint: endpoint{address: "170.170.170.170", port: 0x1234}, pid: 0xCCCCCCCC},
+				{endpoint: endpoint{address: "170.170.170.170", port: 0x1234}, pid: int(pid)},
 			}, false},
 		{"Two entries (IPv6)", IPv6,
 			"02000000" +

